lib/features/recommendation/widgets/ask_rec_button.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/features/recommendation/cubit/recommendation_cubit.dart';
+
+class AskMyParentsButton extends StatelessWidget {
+  const AskMyParentsButton({required this.completed, super.key});
+  final bool completed;
+  @override
+  Widget build(BuildContext context) {
+    final String kidId = context.read<ProfilesCubit>().state.activeProfile.id;
```

Comment:
super tiny comment also regarding naming - shall we keep consistency and name such vars either child (children) or kid (kids)?